airtime_mvc/application/forms/SmartBlockCriteria.php

```diff
@@ -446,6 +446,14 @@ public function startForm($p_blockId, $p_isValid = false)
         }
         $this->addElement($repeatTracks);
 
+        $overflowTracks = new Zend_Form_Element_Checkbox('sp_overflow_tracks');
+        $overflowTracks->setDecorators(array('viewHelper'))
+            ->setLabel(_('Overflow Time Limit:'));
+        if (isset($storedCrit["overflow_tracks"])) {
+            $overflowTracks->setChecked($storedCrit["overflow_tracks"]["value"] == 1?true:false);
```

Comment:
```suggestion
            $overflowTracks->setChecked($storedCrit["overflow_tracks"]["value"] === 1);
```

The `$storedCrit["overflow_tracks"]["value"] == 1` comparison is already a boolean so the `?true:false` isn't needed. It's also likely that the comparison should be `===` rather than `==`.

Reply:
I tried it with == and it worked, with === it was not working, so I assume that the type gets translated at some point by Zend or the interface but it did work without the extra ?true:false so I removed that.

Reply:
`$storedCrit["overflow_tracks"]["value"]` is probably a string that get cast to a boolean, so the `==` is fine in this case (i didn't check the actual type while reviewing). While it can be confusing this kind of auto-type-casting is something I consider as a neat part of PHP :)